language/tools/move-unit-test/src/lib.rs

```diff
@@ -74,6 +74,20 @@ fn format_module_id(module_id: &ModuleId) -> String {
 }
 
 impl UnitTestingConfig {
+    /// Create a unit testing config for use with `register_move_unit_tests`
+    pub fn default_with_bound(bound: Option<u64>) -> Self {
+        Self {
+            instruction_execution_bound: bound.unwrap_or(5000),
```

Comment:
might want to make clear this is a temporal bound, not mem, or some other dimension.

Reply:
huh, it is "instructions" cool.